### PR TITLE
Adds Gradio App example

### PR DESCRIPTION
**Pull request by bradleymarques**
Adds an example Gradio App that people can adapt to run their own Gradio Apps on Highwind.

### CODE REVIEW

File: gradio-app-highwind-example/demo.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        fn=classify_flower,  # There is no need to pass the Request - this is passed automatically
+        inputs=[sepal_length, sepal_width, petal_length, petal_width],
+        outputs=flower_type,
+    )
+
+    # We include a logout button for when the Gradio App is hosted on Highwind. This
+    # button does nothing when running this app locally.
+    gr.Button("Logout", link="/logout")
+
+# You can uncomment the flowing line in order to test the Gradio App locally before
```
Comment (bradleymarques) [inline gradio-app-highwind-example/demo.py:113]:
```suggestion
# You can uncomment the following line in order to test the Gradio App locally before
```

File: gradio-app-highwind-example/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Gradio App Highwind Example
+
+This example shows how to create a [Gradio App](https://www.gradio.app/) and deploy it
+using Highwind.
+
+## Step 1: Create a Gradio App Asset
+
+1. On Highwind, create a new Asset and select "Gradio App" as the Asset type.
+2. Upload a single python file that follows the format of `demo.py` in this folder.
```
Comment (bradleymarques) [inline gradio-app-highwind-example/README.md:9]:
```suggestion
2. Upload a single Python file that follows the format of `demo.py` in this folder.
```

File: gradio-app-highwind-example/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+gradio = "^4.31.4"
+highwind = "1.0.0"
```
Comment (ruankie) [inline gradio-app-highwind-example/requirements.txt:2]: @bradleymarques these don't seem to work when running `pip install -r requirements.txt`.

Changing it to this worked for me:
```suggestion
gradio>=4.31.4
highwind==1.0.0
```

**Review by ruankie — APPROVED** [review]
LGTM thanks @bradleymarques 👍 
Just a minor format change required in `requirements.txt`

**Pull request merged**